web_client/utils/load-model.jsx

```diff
@@ -0,0 +1,16 @@
+import { Promise } from './promise';
+
+const loadModel = (id, Model) => (
+  new Promise((resolve, reject) => {
+    let result = new Model({ _id: id });
+    let req = result.fetch();
+    req.error((payload) => {
+      let e = new Error();
+      e.payload = payload;
+      reject(e);
+    });
+    req.then(() => { resolve(result); });
```

Comment:
I don't actually want to return the value of `resolve(result)` (whatever it is).